src/main/java/net/dv8tion/jda/core/entities/Invite.java

```diff
@@ -367,4 +390,59 @@ default String getURL()
          */
         Set<String> getFeatures();
     }
+
+    /**
+     * POJO for the group information provided by an invite.
+     *
+     * @see #getChannel()
+     */
+    interface Group extends ISnowflake
+    {
+        /**
+         * The icon id of this group.
+         *
+         * @return The group's icon id
+         *
+         * @see    #getIconUrl()
+         */
+        String getIconId();
+
+        /**
+         * The icon url of this group.
+         *
+         * @return The group's icon url
+         *
+         * @see    #getIconId()
+         */
+        String getIconUrl();
+
+        /**
+         * The name of this group.
+         *
+         * @return The group's name
+         */
+        String getName();
+
+        /**
+         * The names of all users in this group. If the users were not included in the
+         * invite, this will return {@code null}. Users will usually only be returned when resolving the invite via the
+         * {@link #resolve(net.dv8tion.jda.core.JDA, java.lang.String, boolean) Invite.resolve()} method with the
+         * {@code withCounts} boolean set to {@code true}.
+         *
+         * @return The names of the groups's users or null if not preset in the invite
+         */
+        List<String> getUsers();
+    }
+
+    /**
+     * Enum representing the type of an invite.
+     *
+     * @see #getType()
+     */
+    enum Type
```

Comment:
InviteType? I wouldn't use the name `Type` because it is also used by java reflections

Reply:
You can already uniquely reference it by using `Invite.Type`. Having `Invite.InviteType` just feels wrong imo.